tst/standard/properties.tst

```diff
@@ -941,7 +941,7 @@ gap> IsMonogenicSemigroup(S);
 true
 gap> x := MinimalSemigroupGeneratingSet(S)[1];;
 gap> S := Semigroup(x, x, x);;
-gap> HasIsMonogenicSemigroup(S) and IsMonogenicSemigroup(S);
+gap> IsMonogenicSemigroup(S);
```

Comment:
This isn't going to cause any problems, but I can't work out why the immediate method on `gap/attributes/properties.gi:771` no longer sets `IsMonogenicSemigroup` to be true for `S`. (Whether that immediate method is a good idea is a separate issue).

Reply:
Me neither, but as you say it is unlikely to cause any problems. Thanks!